setup.py

```diff
@@ -68,7 +68,7 @@
         ],
         extras_require={
             'full': [
-                'finetuner-commons==0.10.1',
+                'finetuner-commons==0.0.1b8',
```

Comment:
did you push this to pypi? then tests should be working right?

Reply:
some of the metadata is missing when validating, such as `builder`, so it will fail until release finished